apps/rush-lib/src/api/VersionPolicyConfiguration.ts

```diff
@@ -31,6 +32,30 @@ export interface IIndividualVersionJson extends IVersionPolicyJson {
   lockedMajor?: number;
 }
 
+/**
```

Comment:
I'm torn here because you can't use `'original'` in combination with a different value. You can either use `'original', 'original'` or you can use `'exact', 'wildcard'`, but you can't use `'exact', 'original'` without a lot of gymnastics.